sim/rogue/poisons.go

```diff
@@ -264,37 +264,26 @@ func (rogue *Rogue) makeInstantPoison(procSource PoisonProcSource) *core.Spell {
 		60: 11340,
 	}[rogue.Level]
 
-	isShivProc := procSource == ShivProc
-
 	hasDeadlyBrew := rogue.HasRune(proto.RogueRune_RuneDeadlyBrew)
 
 	return rogue.RegisterSpell(core.SpellConfig{
 		ActionID:    core.ActionID{SpellID: spellID, Tag: int32(procSource)},
 		SpellSchool: core.SpellSchoolNature,
 		ProcMask:    core.ProcMaskWeaponProc,
+		Flags:       SpellFlagPoison,
 
 		DamageMultiplier: rogue.getPoisonDamageMultiplier(),
 		CritMultiplier:   rogue.SpellCritMultiplier(),
 		ThreatMultiplier: 1,
 
-		// Shiv gives 100% hit to always hit (except for 1% miss of magic table)
-		BonusHitRating: core.TernaryFloat64(isShivProc, 100*core.SpellHitRatingPerHitChance, 0),
```

Comment:
Based on testing against high level mobs, the application of an applied poison does not fail to apply other than the normal 1% to miss that can never be removed. The prime example of this is a level 12 bear fully resisting application of an applied poison which is only explained by magic hit table. Similarly, Poison Knife (using the same mechanic as Shiv) never failed to apply the applied poison (other than a very rare 1% or so) despite the target being level 50+. 

When testing with no poison applied and Deadly Brew engraved poison against a 50+ mob poison applications were frequently resisted and notably different in rate compared to when the same poison was applied to the weapon.

Reply:
I'll add this to all Shiv-Procced poisons then.